### DISCUSSION TITLE
征集好看的截图

### DISCUSSION THREAD

**Discussion by volatile-static** [volatile-static/Chartero#24] [Show and tell]
大家觉得炫酷的图片可以在这里分享，获赞多的入选readme

**Comment by neva2022** [comment]
![1](https://user-images.githubusercontent.com/111333151/217464543-449a435d-f315-46c0-a167-76a4146b438d.svg)
![2](https://user-images.githubusercontent.com/111333151/217464575-32af374d-8598-4e29-8c80-98953f48f212.svg)
支持作者！

**Reply by volatile-static** [reply]
感谢分享！
